packages/components/descriptions/src/description.vue

```diff
@@ -33,6 +33,7 @@ import ElDescriptionsRow from './descriptions-row.vue'
 import { descriptionsKey } from './token'
 import { descriptionProps } from './description'
 
+import { COMPONENT_NAME } from './description-item'
```

Comment:
This line should be moved up, leaving a blank line at line 36, separating the variable and the type.